test/conformance/chainsaw/validate/clusterpolicy/cornercases/two-rules-with-different-action/kyverno-configmap-assert.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kyverno
+  namespace: kyverno
+data:
+  generateSuccessEvents: "true"
```

Comment:
For these changes, can you either use server side apply or kubectl patch to ensure that the existing entries in the config map dont get overwritten by accident